conf/distro/bisdn-linux.conf

```diff
@@ -18,7 +18,7 @@ FEEDNAMEPREFIX ??= "${DISTRO_VERSION}"
 FEEDURIPREFIX  ??= "nightly_builds/${MACHINE}/master/packages_latest-build"
 FEEDDOMAIN     ??= "http://repo.bisdn.de"
 
-DISTRO_FEATURES_DEFAULT ?= "acl argp ext2 ipv4 ipv6 largefile nfs pci pcmcia systemd usbgadget usbhost virtualization xattr"
+DISTRO_FEATURES_DEFAULT ?= "acl argp ext2 ipv4 ipv6 largefile nfs pci pcmcia seccomp systemd usbgadget usbhost virtualization xattr"
```

Comment:
This seems unrelated to the CSP support?

Reply:
Looking into this, this was changed with https://git.yoctoproject.org/meta-virtualization/commit/?h=kirkstone&id=51833d0e131155aeae563c08b892854265bd9a10, which only updated defaults, so seccomp is only a soft dependency not a hard one.

I suggest instead setting `PACKAGECONF` for docker-ce to the old value, to restore the old state.

Reply:
Addressed by https://github.com/bisdn/meta-switch/pull/131